assets/themes/solarized.tmTheme

```diff
@@ -174,6 +174,18 @@
                     <string>#21000000</string>
                 </dict>
             </dict>
+            <!-- FIXME: does this color fit the theme? -->
+            <dict>
+                <key>name</key>
+                <string>Error</string>
+                <key>scope</key>
+                <string>error</string>
+                <key>settings</key>
+                <dict>
+                    <key>foreground</key>
+                    <string>#01000000</string>
```

Comment:
I'm not sure what color would work best. But since this is rarely shown, defaulting to Ansi-red sound good 👍